clients/web/src/common/api.js

```diff
@@ -1,18 +1,26 @@
+import groupBy from 'lodash/groupBy'
+
 import Answer from './Answer'
 import Assessment from './Assessment'
 import Benchmark from './Benchmark'
 import Organization from './Organization'
 import OrganizationGroup from './OrganizationGroup'
 import Question from './Question'
 import Score from './Score'
+import Section from './Section'
+import Subcategory from './Subcategory'
 import { getPracticeList } from './Practice'
 import { getShareEntryList } from './ShareEntry'
-import { VALID_ASSESSMENT_STEPS } from '../config/app'
+import {
+  MAP_QUESTION_FORM_TYPES,
+  QUESTION_COMMENT_TYPE,
+  QUESTION_ENERGY_CONSUMED,
+  QUESTION_WATER_CONSUMED,
+  QUESTION_WASTE_GENERATED,
+} from '../config/app'
 
-const API_BASE_URL = process.env.VUE_APP_API_BASE_URL || (
-  process.env.VUE_APP_STANDALONE
-  ? `${window.location.origin}/envconnect/api`
-  : `${process.env.BASE_URL}api`)
+const API_HOST = process.env.VUE_APP_STANDALONE ? window.location.origin : ''
+const API_BASE_URL = `${API_HOST}${process.env.VUE_APP_ROOT}${process.env.VUE_APP_API_BASE}`
```

Comment:
Can we keep `process.env.VUE_APP_API_BASE_URL` as the `API_BASE_URL` if it is defined?
It is way too complicated trying to figure out how to create the expected `API_BASE_URL` out of `API_HOST`, `VUE_APP_ROOT` and `VUE_APP_API_BASE` while insuring the web client still generates its own expected URLs.

Reply:
Per https://github.com/djaodjin/envconnect/pull/346/commits/12cec396c3601699aa4c861fa74cbeaeb2fde49f, I realized that the strings that made up `VUE_APP_API_BASE_URL` were used at different parts of the app that's why I split it up and moved these strings to `.env`. I thought creating another `env` variable made up of `${process.env.VUE_APP_ROOT}` and `${process.env.VUE_APP_API_BASE}` would be overkill. Do you think it would help to add it or add some documentation around these values?